# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1290481668 28800
#      Mon Nov 22 19:07:48 2010 -0800
# Node ID 646063a22a35cddf4699f71c25118b8a98ef1818
# Parent  f44c479592560f9f43db3c5bde9098367c15cfd2
sqp.m: Use correct stopping tolerance in documentation.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-11-21  Rik  <octave@nomad.inbox5.com>
+
+	* optimization/sqp.m: Use correct stopping tolerance in documentation.
+
 2010-11-21  Kai Habel  <kai.habel@gmx.de>
 
 	* (plot/uigetdir.m, plot/private/__fltk_file_filter__.m:): Check if
 	file argument is a valid path, don't use fileparts in this case.
 
 2010-11-21  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Fix plot titles so they are visible for
diff --git a/scripts/optimization/sqp.m b/scripts/optimization/sqp.m
--- a/scripts/optimization/sqp.m
+++ b/scripts/optimization/sqp.m
@@ -144,17 +144,17 @@
 ## be a scalar in which case all elements of @var{x} will share the same
 ## bound.  If only one bound (lb, ub) is specified then the other will 
 ## default to (-@var{realmax}, +@var{realmax}).
 ##
 ## The seventh argument specifies the maximum number of iterations.
 ## The default value is 100.
 ##
 ## The eighth argument specifies the tolerance for the stopping criteria.
-## The default value is @code{eps}.
+## The default value is @code{sqrt(eps)}.
 ##
 ## The value returned in @var{info} may be one of the following:
 ##
 ## @table @asis
 ## @item 101
 ## The algorithm terminated normally.  
 ## Either all constraints meet the requested tolerance, or the stepsize,
 ## @tex
